pathexp/pathexp.go

```diff
@@ -341,16 +377,31 @@ func (pe *PathExp) Org() string {
 	return string(pe.org)
 }
 
+// MatchProject identifies if the given string matches the project value
+func (pe *PathExp) MatchProject(str string) bool {
+	return matchSegment(pe.project, str)
+}
```

Comment:
Yeah I didn't want to make all the fields public (since they weren't before) just to faciliatate compare.